src/Nethermind/Nethermind.Facade/Proxy/Models/MultiCall/MultiCallCallResult.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Facade.Proxy.Models.MultiCall;
+
+public class MultiCallCallResult
+{
+    public ResultType Type
+    {
+        get
+        {
+            if (Error != null) return ResultType.Failure;
+
+            if (Logs != null) return ResultType.Success;
+
+            return ResultType.Invalid;
```

Comment:
`Error` is not null in `CallResultInvalid` as well. Given the spec, I would do it like this:
```suggestion
            if (Error is null) return ResultType.Success;

            if (Return is not null) return ResultType.Failure;

            return ResultType.invalid;
```

Reply:
Ah sorry, made a mistake here with capitlization